src/Display/Progress/Progress.tsx

```diff
@@ -71,11 +73,15 @@ const Progress: React.FunctionComponent<Props> = (props) => {
         <ProgressContent tabIndex={-1} size={sizeInEm}>
           <svg width="8em" height="8em" viewBox="0 0 100 100">
             <circle cx="50" cy="50" r="45" fill="none" stroke={SecondaryColor.lighterblack} strokeWidth="8" />
-            <circle className="progress-circle__value" cx="50" cy="50" r="45" fill="none" stroke={color} strokeWidth="8" />
+            <circle className="progress-circle-value" aria-label="progress-circle-value" cx="50" cy="50" r="45" fill="none" stroke={color} strokeWidth="8" />
           </svg>
           <ProgressLabelWrapper aria-hidden="true">
-            {content || <PercentageCompletion>{`${normalizedPercentage > 100 ? 100 : normalizedPercentage}%`}</PercentageCompletion>}
-            {content ? null : <LabelText>COMPLETE</LabelText>}
+            {content || (
+              <PercentageCompletion aria-label="percentage-completion">
+                {`${normalizedPercentage > 100 ? 100 : normalizedPercentage}%`}
```

Comment:
@franciscrispin given your checks above, normalizedPercentage will always be below 100 ya? But in the spirit of defensive programming, this redundant check is good. However, following this line of programming, how about checks for <0 values?

Reply:
Oh no I didn't realise it was duplicated. Hmm if it's redundant why would it still be good? @seahyc

Reply:
Decided to remove it!